### ISSUE TITLE
Accessibility Warning: DialogContent Missing Description or aria-describedby

### ISSUE THREAD

**Issue by modamaan** [coronasafe/care_fe#10617]
**Describe the bug**
When using DialogContent, the console logs a warning about a missing Description or aria-describedby={undefined}. This issue may affect accessibility by not providing a proper description for assistive technologies.

**To Reproduce**

[Click here to reproduce](https://care.ohc.network/facility/605215bd-6215-45dd-bc20-aaeceb474ca1/settings/general)
See error

**Expected behavior**
DialogContent should not trigger accessibility warnings in the console. It should either automatically provide a valid aria-describedby attribute or allow users to specify a description to ensure proper accessibility compliance.

**Screenshots**

https://github.com/user-attachments/assets/0d010bc2-e726-4739-a050-a35d0b828f62

**Desktop (please complete the following information):**

- OS: [windows]
- Browser [chrome]
- Version [132.0.6834.111]

**Additional context**

**Comment by modamaan** [comment]
@rithviknishad  @Jacobjeevan  if it is valid, i would like to take it up

**Comment by Jacobjeevan** [comment]
Let's not add anything automatically, add the appropriate attributes for each of the dialog content uses.
